release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -916,3 +916,26 @@ fail.
 * The public cloud access image for RHEL cannot be used directly if you enable FIPS mode. This is caused by public cloud images not allowing kernel integrity checks. To do this, you must upload your own images. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1788051[*BZ#1788051*])
 * The Operator Lifecycle Manager (OLM) does not work in {product-title} when Kuryr SDN is enabled. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1786217[*BZ#1786217*])
 * The `oc adm catalog build` and `oc adm catalog mirror` commands do not work for the restricted cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1773821)[*BZ#1773821*])
+
+* When upgrading a {product-title} cluster from  4.1 to 4.2 to 4.3, there is a
+possibility that the Node Tuning Operator tuned pods can get stuck in
```

Comment:
Suggestion: adding `the` in front of `ContainerCreating state` may offer a more natural flow.

Reply:
s/pods/Pods